reports/Oregon.md

```diff
@@ -6747,6 +6747,36 @@ geolocation: 45.4928304, -122.6729251
 * https://twitter.com/MrOlmos/status/1313748616171847680
 
 
+### Police make violent mass arrests | October 10th
+
+15 minutes into a protest near the Blazers Boys and Girls Club, police violently arrest protesters en masse, tackling some protesters and deploying mace against others. Police also ripped umbrellas away from protesters, destroying them. Police had not declared an unlawful assembly, although police had allegedly issued a single warning to protesters to stay off the streets. Several journalists were shoved by police and forced disperse, in clear violation of the recently renewed TRO exempting members of the press from dispersal orders.
```

Comment:
by police and forced disperse

Should be

by police and forced **to** disperse

Reply:
fixed